### PR TITLE
🌐 Add Dutch translation for `docs/nl/docs/index.md`

### PR DESCRIPTION
**Pull request by maxscheijen**
Relates to #4701

This translates`docs/index.md` into Dutch.

### CODE REVIEW

**Comment by Kludex** [comment]
I guess it's duplicated of #4770 😟

A review there will probably be welcome 👀

File: docs/nl/docs/index.md [hunk 1]
```diff
@@ -296,106 +291,106 @@ def update_item(item_id: int, item: Item):
...
-    * All this would also work for deeply nested JSON objects.
-* Convert from and to JSON automatically.
-* Document everything with OpenAPI, that can be used by:
-    * Interactive documentation systems.
-    * Automatic client code generation systems, for many languages.
-* Provide 2 interactive documentation web interfaces directly.
+Terugkomend op het vorige code voorbeeld, **FastAPI** zal:
+
+* Valideren dat er een `item_id` bestaat in het pad voor `GET` en `PUT` verzoeken.
+* Valideren dat `item_id` van het type `int` is voor `GET` en `PUT` verzoeken.
```
Comment (tony1658) [inline docs/nl/docs/index.md:373]: Doesn't Validate that the `item_id` translate to Valideren dat **de** `ìtem_id`?

File: docs/nl/docs/index.md [hunk 2]
```diff
@@ -413,56 +408,56 @@ Try changing the line with:
...
     * **CORS**
     * **Cookie Sessions**
-    * ...and more.
+    * ...en meer.
 
-## Performance
+## Prestaties
 
-Independent TechEmpower benchmarks show **FastAPI** applications running under Uvicorn as <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">one of the fastest Python frameworks available</a>, only below Starlette and Uvicorn themselves (used internally by FastAPI). (*)
+nafhankelijke TechEmpower benchmarks tonen **FastAPI** applicaties draaiend onder Uvicorn als <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">een van de snelste Python frameworks beschikbaar</a>, alleen onder Starlette en Uvicorn zelf (intern gebruikt door FastAPI). (*)
```
Comment (tony1658) [inline docs/nl/docs/index.md:434]: Do you mean **Onafhankelijke** instead?

File: docs/nl/docs/index.md [hunk 3]
```diff
@@ -413,56 +408,56 @@ Try changing the line with:
...
-* <a href="https://andrew-d.github.io/python-multipart/" target="_blank"><code>python-multipart</code></a> - Required if you want to support form <abbr title="converting the string that comes from an HTTP request into Python data">"parsing"</abbr>, with `request.form()`.
-* <a href="https://pythonhosted.org/itsdangerous/" target="_blank"><code>itsdangerous</code></a> - Required for `SessionMiddleware` support.
-* <a href="https://pyyaml.org/wiki/PyYAMLDocumentation" target="_blank"><code>pyyaml</code></a> - Required for Starlette's `SchemaGenerator` support (you probably don't need it with FastAPI).
+* <a href="https://requests.readthedocs.io" target="_blank"><code>requests</code></a> - Vereist als je de `TestClient` wilt gebruiken.
+* <a href="https://jinja.palletsprojects.com" target="_blank"><code>jinja2</code></a> - Vereist als u de standaard templateconfiguratie wilt gebruiken.
+* <a href="https://andrew-d.github.io/python-multipart/" target="_blank"><code>python-multipart</code></a> - Vereist als je het <abbr title="chet omzetten van de string die uit een HTTP verzoek komt in Python data">"parsen"</abbr> van formulieren wil ondersteunen met `requests.form()`.
```
Comment (tony1658) [inline docs/nl/docs/index.md:449]: Change `chet omzetten` to `het omzetten van`

File: docs/nl/docs/index.md [hunk 4]
```diff
@@ -413,56 +408,56 @@ Try changing the line with:
...
-Used by FastAPI / Starlette:
+Gebruikt door FastAPI / Starlette:
 
-* <a href="https://www.uvicorn.org" target="_blank"><code>uvicorn</code></a> - for the server that loads and serves your application.
-* <a href="https://github.com/ijl/orjson" target="_blank"><code>orjson</code></a> - Required if you want to use `ORJSONResponse`.
+* <a href="https://www.uvicorn.org" target="_blank"><code>uvicorn</code></a> - voor de server die uw applicatie laadt en bedient.
+* <a href="https://github.com/ijl/orjson" target="_blank"><code>orjson</code></a> - Vereist als je `ORJSONResponse` wilt gebruiken.
 
-You can install all of these with `pip install "fastapi[all]"`.
+U kunt deze allemaal installeren met `pip install "fastapi[all]"`.
```
Comment (tony1658) [inline docs/nl/docs/index.md:459]: You are using `je` and `u` isn't it better to either choose the formal or informal version instead for consistency?

**Review by tyzh-dev — CHANGES_REQUESTED** [review]
A couple typos and inconsistency with combining informal and formal writing.

**Comment by tony1658** [comment]
> I guess it's duplicated of #4770 😟
> 
> A review there will probably be welcome 👀

That one translates the tutorial instead of the index page. So, not a duplicate but yeah I can take a look.

**Comment by tiangolo** [comment]
It seems this was handled in another PR, so I'm gonna close this one, but thanks for the effort! :coffee:

**Pull request closed without merging**